pkg/base/capacityplanner/planner.go

```diff
@@ -160,38 +209,47 @@ func (cm *CapacityManager) selectCapacityOnNode(ctx context.Context, node string
 	return result
 }
 
-func (cm *CapacityManager) update(ctx context.Context) error {
-	logger := util.AddCommonFields(ctx, cm.logger, "CapacityManager.update")
-	cm.nodesCapacity = map[string]*nodeCapacity{}
-	capacity, err := cm.capReader.ReadCapacity(ctx)
-	if err != nil {
-		logger.Errorf("Failed to read capacity: %s", err.Error())
-		return err
+func (cm *CapacityManager) isAnotherACRWithLVGReserved(ctx context.Context, volumes []*genV1.Volume, acrs []acrcrd.AvailableCapacityReservation) bool {
+	logger := util.AddCommonFields(ctx, cm.logger, "CapacityManager.isAnotherACRWithLVGReserved")
+
+	// check if current ACR has no LVG volumes
```

Comment:
ACR has reservation requests not volumes.

Reply:
updated